app/models/user_profile.rb

```diff
@@ -0,0 +1,58 @@
+# == Schema Information
+#
+# Table name: user_profiles
+#
+#  id                       :bigint           not null, primary key
+#  address_city             :string
+#  address_country          :string
+#  address_country_code     :string
+#  address_county           :string
+#  address_lat              :decimal(8, 6)
+#  address_line1            :string
+#  address_line2            :string
+#  address_long             :decimal(9, 6)
+#  address_state            :string
+#  address_zip              :string
+#  birthday                 :date
+#  custom_fields            :jsonb
+#  facebook                 :string
+#  first_name               :string
+#  instagram                :string
+#  last_name                :string
+#  phone_number             :string
+#  profile_completed        :boolean          default(FALSE)
+#  subscribed_to_newsletter :boolean
+#  title                    :string
+#  twitter                  :string
+#  website                  :string
+#  created_at               :datetime         not null
+#  updated_at               :datetime         not null
+#  user_id                  :bigint
+#
+# Indexes
+#
+#  index_user_profiles_on_profile_completed  (profile_completed)
+#  index_user_profiles_on_user_id            (user_id)
+#
+class UserProfile < ApplicationRecord
+  TITLES = ["Mr", "Mrs", "Ms", "Miss", "Dr", "Mx", "Profesor"]
+  PHONE_NUMBER_REGEX = /\A\+[1-9]\d{1,14}\z/
+
+  belongs_to :user
+
+  validates :title, inclusion: {in: TITLES}, allow_nil: true
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :birthday, inclusion: {in: ->(date) { 15.years.ago..Date.today }}, allow_nil: true
```

Comment:
Why not to validate > 18? Like you must be an adult to join. It is not the case?

Reply:
I'm not sure TBH, I don't think we will have members less than 18 years old, but I'm adding this as a more general validation. Not as a way to prevent people from signing up.